tests/store/tracking/test_file_store.py

```diff
@@ -542,10 +542,13 @@ def test_delete_restore_run(self):
         _, run_dir = fs._find_run_root(run_id)
         # Should not throw.
         assert fs.get_run(run_id).info.lifecycle_stage == "active"
+        # Verify that run deletion is idempotent by deleting twice
```

Comment:
Did you want to issue 2 delete commands here?

Reply:
Indeed! Fixed :)

Reply:
Great catch! Forgot to remove `check_run_is_active()` from `FileStore.delete_run()`.